ResearchKit/Consent/ORKConsentSection+AssetLoading.m

```diff
@@ -42,6 +42,10 @@ OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 NSURL *ORKMovieURLForConsentSectionType(ORKConsentSectionType type) {
     CGFloat scale = [[UIScreen mainScreen] scale];
     
+    // For iPad, use the movie for the next scale up
+    if(UI_USER_INTERFACE_IDIOM() == UIUserInterfaceIdiomPad && scale < 3)
```

Comment:
This needs curly brackets around the block and a space between if and open paren.